astropy/stats/bayesian_blocks.py

```diff
@@ -366,23 +367,23 @@ def fit(self, t, x=None, sigma=None):
 
             # T_k: width/duration of each block
             if 'T_k' in self._fitness_args:
-                kwds['T_k'] = block_length[:R + 1] - block_length[R + 1]
+                kwds['T_k'] = block_length[: R + 1] - block_length[R + 1]
```

Comment:
@WilliamJamieson would this be better as `[: (R + 1)]`?

Reply:
See e9adf9e.

Reply:
thanks